src/main/java/com/notably/logic/parser/NotablyParser.java

```diff
@@ -48,6 +53,8 @@ public NotablyParser(Model notablyModel) {
      * @throws ParseException when there is a invalid input string.
      */
     public List<? extends Command> parseCommand(String userInput) throws ParseException {
+        requireNonNull(userInput);
+        logger.info(String.format("Parsing '%s'", userInput));
```

Comment:
Perhaps you can add more logging after this one.

1. If parsing fails, you can log something
2. If parsing successful, you can log something else

It's also better if you can be more specific about the kind of errors/success you've encountered! At the end of the day, that's when the usefulness of logging starts to show!

Reply:
Got it will log the errors inside the respective `CommandParsers` so that the logger can track the exact location where the error is found.